dev/tools/test/fake_process_manager.dart

```diff
@@ -0,0 +1,185 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:process/process.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+class FakeProcessManager extends Mock implements ProcessManager {
```

Comment:
nit: add a doc comment?

Reply:
Good point. I added doc comments for the entire file.